update-dois/Jenkinsfile

```diff
@@ -0,0 +1,120 @@
+import groovy.json.JsonSlurper
+
+pipeline {
+	agent any
+
+	stages {
+		stage('Check if upstream builds succeeded'){
+			steps{
+				script{
+					def updateStIdsStatusUrl = httpRequest authentication: 'jenkinsKey', url: "${env.JENKINS_JOB_URL}/job/${env.RELEASE_NUMBER}/job/UpdateStableIdentifiers/lastBuild/api/json"
+					def updateStIdsStatusJson = new JsonSlurper().parseText(updateStIdsStatusUrl.getContent())
+					if(updateStIdsStatusJson['result'] != "SUCCESS"){
+						error("Most recent UpdateStableIdentifiers build status: " + updateStIdsStatusJson['result'])
+					}
+				}	
+		    	}
+	    	}
+		stage('Setup: Back up DBs'){
+			steps{
+				script{
+					dir('update-dois'){
+						withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
+							sh "mysqldump -u$user -p$pass ${env.RELEASE_CURRENT} > ${env.RELEASE_CURRENT}_${env.RELEASE_NUMBER}_before_update_dois.dump"
+							sh "gzip -f ${env.RELEASE_CURRENT}_${env.RELEASE_NUMBER}_before_update_dois.dump"
+							sh "mysqldump -u$user -p$pass -h${env.CURATOR_SERVER} ${env.GK_CENTRAL} > ${env.GK_CENTRAL}_${env.RELEASE_NUMBER}_before_update_dois.dump"
+							sh "gzip -f ${env.GK_CENTRAL}_${env.RELEASE_NUMBER}_before_update_dois.dump"
```

Comment:
Since it's re-used a few times, maybe `${env.GK_CENTRAL}_${env.RELEASE_NUMBER}_before_update_dois.dump` could be assigned to its own local variable if that's possible.